doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -114,8 +126,9 @@ Now create the logical volumes for ``block``:
    lvcreate -l 100%FREE -n block-2 ceph-block-2
    lvcreate -l 100%FREE -n block-3 ceph-block-3
 
-We are creating 4 OSDs for the four slow spinning devices, so assuming a 200GB
-SSD in ``/dev/sdx`` we will create 4 logical volumes, each of 50GB:
+Because there are four rotational devices, there will be four OSDs. Supposing
```

Comment:
If someone ever uses a DVD-RAM drive for an OSD we can and should slap them, so let's call them HDDs consistently.

Reply:
Accepted.